pkgs/applications/editors/vim/vimacs.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, config, vim_configurable, macvim, vimPlugins
+, useMacvim ? stdenv.isDarwin && (config.vimacs.macvim or true)
+, vimacsExtraArgs ? "" }:
+
+stdenv.mkDerivation rec {
+  name = "vimacs-${vimPlugins.vimacs.version}";
```

Comment:
You should set `pname` and `version` instead. The `vim` version is probably more relevant than the `vimacs` version, or when in doubt you could use both. But the `vim` version should definitely show up.

Reply:
Done